commands/unfollow.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  const host = require('../lib/host')
+  const util = require('../lib/util')
+  const fetcher = require('../lib/fetcher')(heroku)
+  let {app, args, flags} = context
+  let db = yield fetcher.addon(app, args.database)
+
+  if (!app) app = db.app.name
+
+  let replica = yield heroku.get(`/client/v11/databases/${db.name}`, {host: host(db)})
+
+  if (!replica.following) throw new Error(`${cli.color.addon(db.name)} is not a follower`)
+
+  let origin = util.databaseNameFromUrl(replica.following, yield heroku.get(`/apps/${app}/config-vars`))
+  yield cli.confirmApp(app, flags.confirm, `WARNING: Destructive action
+${cli.color.addon(db.name)} will become writeable and no longer follow ${origin}. This cannot be undone.
+`)
+
+  yield cli.action(`${cli.color.addon(db.name)} unfollowing`, co(function * () {
+    yield heroku.put(`/client/v11/databases/${db.name}/unfollow`, {host: host(db)})
+  }))
+}
+
+module.exports = {
+  topic: 'pg',
+  command: 'unfollow',
+  description: 'stop a replica from following and make it a read/write database',
```

Comment:
Maybe `...make it a writeable database`? That's the interesting part here, and I doubt leaving the 'read' part implicit is going to confuse anyone.